### PR TITLE
ACM-8243: update hosted cluster destroy docs

### PR DESCRIPTION
**Pull request by rokej**
I updated the hosted cluster destroy doc.

### CODE REVIEW

File: clusters/hosted_control_planes/destroying_hosted_bare_metal.adoc [hunk 1]
```diff
@@ -7,4 +7,29 @@ You can use the console to destroy bare metal hosted clusters. Complete the foll
 
 . On the _Clusters_ page, select the cluster that you want to destroy.
 
-. In the *Actions* menu, select *Destroy clusters* to remove the cluster.
\ No newline at end of file
+. In the *Actions* menu, select *Destroy clusters* to remove the cluster.
+
+[#hypershift-cluster-destroy-bm-cli]
+= Destroying a hosted cluster on bare metal using command line
```
Comment (lahinson) [inline clusters/hosted_control_planes/destroying_hosted_bare_metal.adoc]:
```suggestion
== Destroying a hosted cluster on bare metal by using the command line
```

File: clusters/hosted_control_planes/destroying_hosted_bare_metal.adoc [hunk 2]
```diff
@@ -7,4 +7,29 @@ You can use the console to destroy bare metal hosted clusters. Complete the foll
...
+
+. Delete the managed cluster resource on {mce-short} by running the following command:
+
++
+----
+oc delete managedcluster <cluster_name>
+----
+
++
+Replace `cluster_name` with the name of your cluster.
```
Comment (lahinson) [inline clusters/hosted_control_planes/destroying_hosted_bare_metal.adoc]:
```suggestion
where _`cluster_name`_ is the name of your cluster.
```

File: clusters/hosted_control_planes/hypershift_cluster_destroy_aws.adoc [hunk 3]
```diff
@@ -3,22 +3,22 @@
...
 
 +
 ----
-hcp destroy cluster aws --name <cluster_name> --infra-id <infra_id> --aws-creds <aws-credentials> --base-domain <base_domain> --destroy-cloud-resources
+oc delete managedcluster <cluster_name>
 ----
 
 +
-Replace names where necessary.
+Replace `cluster_name` with the name of your cluster.
```
Comment (lahinson) [inline clusters/hosted_control_planes/hypershift_cluster_destroy_aws.adoc]:
```suggestion
where _`cluster_name`_ is the name of your cluster.
```

File: clusters/hosted_control_planes/hypershift_cluster_destroy_kubevirt.adoc [hunk 4]
```diff
@@ -1,10 +1,24 @@
...
 
+. Delete the managed cluster resource on {mce-short} by running the following command:
+
++
+----
+oc delete managedcluster <cluster_name>
+----
+
++
+Replace `cluster_name` with the name of your cluster.
```
Comment (lahinson) [inline clusters/hosted_control_planes/hypershift_cluster_destroy_kubevirt.adoc]:
```suggestion
where _`cluster_name`_ is the name of your cluster.
```

**Review by lahinson — APPROVED** [review]
@rokej I provided a few suggestions for your consideration. Thanks!

**Comment by rokej** [comment]
@lahinson Thank you for the comments. I updated the docs.

**Pull request merged**